### DISCUSSION TITLE
Add Split Tunneling to system tray's context menu

### DISCUSSION THREAD

**Discussion by 6ftClaud** [mullvad/mullvadvpn-app#9075] [Feature requests & Ideas]
## Issue report

Operating system: Arch Linux, kernel version `6.0.5-arch1-1`

App version: `2022.5`

### Issue description
It'd be nice to have `Split Tunneling` added to the system tray context menu. It'd save a few clicks and it's worthwhile when one needs to use that feature often.  
Otherwise, where should I look in order to add it myself? I could fork the project and add it for myself.

**Comment by pizzadude** [comment]
check out "mullvad-exclude" command

for example:

mullvad-exclude steam

**Comment by faern** [comment]
@6ftClaud How do you imagine this feature looking in the tray menu? One sub menu with *all apps* that can be checked or unchecked? It's going to be a very long sub menu.

**Comment by pizzadude** [comment]
@faern 
`One sub menu with all apps that can be checked or unchecked?`
Please never do that. I can't use the "Split Tunneling" part of the Mullvad app because of how long it takes to generate the list of applications, which has very high CPU usage when generating the list also.

**Comment by 6ftClaud** [comment]
I am aware of `mullvad-exclude`'s existence, but this is more about a QoL change which would make the Split Tunneling easier to access in the GUI. And I think less techy people would appreciate not having to open up terminals or editing `.desktop` files to add the command.  
@faern I was thinking along the lines of having a `Split Tunneling` option in the context menu open the Split Tunneling tab in the GUI directly. It's a fairly big feature so I think it would make sense having it in the context menu. Wouldn't need to dig through the GUI to get to it. Yes, it's only a few clicks, but having a context menu option would be infinitely better.

**Comment by MagicLike** [comment]
Another possibility would be to add a new entry into the app menu / start menu with split tunnelling already enabled. Also pinning apps in the split tunnelling menu would improve UX as the users would not have to search for an often split tunnelled app every time.
